src/types/Firestore.ts

```diff
@@ -0,0 +1,68 @@
+import { Venue } from "./Venue";
+import { Purchase } from "./Purchase";
+import { User } from "./User";
+import {
+  RestrictedChatMessage,
+  PrivateChatMessage,
+} from "components/context/ChatContext";
+import { VenueEvent } from "./VenueEvent";
+import { Table } from "./Table";
+import { PartyMapVenue } from "./PartyMapVenue";
+
+interface VenueStatus {
+  currentVenue: boolean;
+  currentEvent: boolean;
+  eventPurchase: boolean;
+  venueChats: boolean;
+  venueEvents: boolean;
+  userPurchaseHistory: boolean;
+}
+
+interface Reaction {
```

Comment:
This looks wrong. Shouldn't it have the same type as in the creation form?

Reply:
indeed